spark/components/tabs/react/SprkTabs.stories.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 import SprkTabs from './SprkTabs';
-import SprkTabsPanel from './SprkTabsPanel';
+import SprkTabsPanel from './components/SprkTabsPanel';
 
 export default {
   title: 'Components|Tabs',
```

Comment:
will these need
```
  decorators: [withKnobs],
  parameters: { component: SprkTabs}
```
So it shows up in the docs page? maybe knobs not needed but the parameters